### ISSUE TITLE
test question

### ISSUE THREAD

**Issue by kkkkkk1234** [Tramac/DenseVoxNet-tensorflow#1]
hello , I am working with this job, and i use this with the data of  the 'training_axial_crop_pat0-label.nii' and  'training_axial_crop_pat0.nii' ,and the number of the data is ten ,after I finished the step of 'python prepare_h5_data.py' and 'python train.py' , it works well ,but unfortunately when I go to the next step ,I  found that it couldn't show  any results ,I want to know if i have any wrong with this ? I sincerely hope I can get your replys , thank you very much!

**Comment by Tramac** [comment]
Hello, are there relevant error logs?

**Comment by kkkkkk1234** [comment]
Hello,Firstly,thank you for the reply.It did't show any error,it just finished，and turn to the next command line without any outputs.So what's wrong with this ?I Followed the steps in Read Me file.Thank you for your help.Looking forward to your next reply!Thanks!

**Comment by Tramac** [comment]
This situation occurred because of the lack of visualization. Firstly, the prediction results  were saved in `fusion_Img_L2` and `vote_label` (both nparray format) in `train.py.` You can save them in any format you want, such as `npy,` `mat,` `nrrd` or others. And then visualize the slice in Python. I am very sorry that I don't have enough time to complete this part now, but I will add it as soon as possible. If there are other questions, please continue to post here.

**Comment by kkkkkk1234** [comment]
Ok , Thank you very much ,I will try this step .And I have another question , when I doing the test step , I can't understand why the label data is still placed in the  data folder?Thank you for your instantly reply!

**Comment by Tramac** [comment]
All data in the `data` folder is used for training. Because the data came from a competition website, where had no public test data, data used in test stage was also from `data` folder, aiming to test the code can run without bugs. In theory, the test data should come from a new data source, which did't appear in the training set. Thank you for your question, you are right!
